app/test/integration_test.go

```diff
@@ -284,3 +284,73 @@ func queryTx(clientCtx client.Context, hashHexStr string, prove bool) (*rpctypes
 
 	return node.Tx(context.Background(), hash, prove)
 }
+
```

Comment:
Considering this integration test suite is no longer strictly for block size, proposal to rename the above skip description to something like:

```diff
-		s.T().Skip("skipping block size integration test in short mode.")
+		s.T().Skip("skipping app/test/integration_test in short mode.")
```

Reply:
4595dd2